ddtrace/contrib/elasticsearch/patch.py

```diff
@@ -45,12 +46,23 @@ def patch():
         _patch(elasticsearch)
 
 
+def _determine_transport_module(elasticsearch):
+    transport_module = getattr(elasticsearch, "transport", False)
```

Comment:
Are there any chances that importing the proper `elasticsearch{1-8}` module does not auto-import the `transport` module as well?

e.g. I am wondering if doing a:

```python
try:
    import elasticsearch.transport as elastic_transport
except ImportError:
    import elastic_transport
```

would be better?

obviously we have to know which elasticsearch module (`elasticsearch`, `elasticsearch7`, `elasticsearch8`, etc) we are using, but wondering if this would be more reliable?

Reply:
I checked all versions of elasticsearch and they all import the transport module in the __init__.py file. So there is no chance that if elasticsearch is imported the transport module is not imported. 

Also, if I am not missing something we could never use this syntax and we would need to use importlib. 

But thank you for your comments 👍